taqtile-desafio/package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "taqtile-desafio",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc -b && vite build",
+    "lint": "eslint .",
+    "preview": "vite preview"
+  },
+  "dependencies": {
+    "react": "^18.3.1",
```

para manter a consistência do projeto durante o tempo, costumamos fixar as versões das dependências
```suggestion
    "react": "18.3.1",
```

---

aqui ainda aparece com `^` nas dependências

---

Resolvindo também.